pages/cloud/dedicated/partitioning_ovh/guide.en-gb.md

```diff
@@ -0,0 +1,294 @@
+>
+
+#### API Partition Layout
+
+##### Structure
+
+A partiton layout is a list of partitions. Here is an example of partition structure:
+
+```JSON
+{
+  "raid": "1",
+  "filesystem": "ext4",
+  "volumeName": "data",
+  "size": 0,
+  "order": 4,
+  "type": "lv",
+  "mountpoint": "/var/lib/vz",
+}
+```
+
+> [!primary]
+>
+> order: is the partition order within the partition array
+>
+
+<!-- markdownlint-disable-next-line MD028 -->
+> [!primary]
+>
+> type: is one of primary, logical, lv
+> lv: means that LVM layers will be added on top of the [software RAID](https://docs.ovh.com/gb/en/dedicated/raid-soft/) device (or the physical partition if RAID level is 0)
+>
+> - primary: only makes sense for MBR partition tables (a few old non-UEFI servers still don't support GPT)
+> - logical: if you don't know what to choose
+>
+
+<!-- markdownlint-disable-next-line MD028 -->
+> [!primary]
+>
+> volumeName: only useful if type lv is set
+>
+
+##### Filesystems & LVM + Raid levels Comptibility Matrix
+
+The following table gives you an overview of filesystem compatibility with their raid levels as well as LVM compatibility within the OVHcloud context:
+
+|Filesystem|LVM|RAID 0|RAID 1|RAID 5|RAID 6|RAID 7|RAID 10|
+|---|---|---|---|---|---|---|---|
+|Btrfs, ext3, ext4, ReiserFS, XFS|✅|✅|✅|✅|✅|✅|✅|
+|ZFS¹|❌|✅|✅|✅|✅|✅|❌|
+|swap|❌|⚠²|❌|❌|❌|❌|❌|
+|NTFS|❌|✅|✅³|❌|❌|❌|❌|
+|UFS, VMFS5, VMFS6, VMFS-L⁴|❌|❌|❌|❌|❌|❌|❌|
+
+¹ See [RAID Z vs standard RAID](#RAIDz2RAID) table for more details.
+² swap raid level can only be defined with a RAID level set to 1 within the [OVHcloud API](https://api.ovh.com/). In reality swap is not a raid. When a swap with size `s` is defined on a server with `n` disks, this will create `n` partitions on every disk of size `s` without any software raid device.
+³ Windows native raid (the one configured by the OVHcloud installer) only supports mirroring that is a raid 1 but only between 2 disks.
+⁴ ESXi installer is not compatible with custom partitioning via the [OVHcloud API](https://api.ovh.com/). Partitioning is defined by the software editor. Nevertheless, [OVHcloud API](https://api.ovh.com/) can give you an idea on how the partitioning looks like: see section [OVHcloud templates](#OVHcloudtemplates) for more details.
+
+> [!warning]
+>
+> This table is only given as information for customer. Please note that LVM and especially filesystems compatibility also depends on the OS (OVHcloud template) customer is installing. See [OVHcloud templates](#OVHcloudtemplates) section for more details.
+>
+
+#### RAID Z vs standard RAID <a name="RAIDz2RAID"></a>
+
+In order to make [OVHcloud API](https://api.ovh.com/) as simple as possible and because RAID Z and only RAID Z is exclusively possible on ZFS, customer must define a standard RAID within the API for ZFS filesystems in order to have RAID Z. The following table gives you the matching between standard RAID and RAID Z as well as a reminder with their respective characteristics.
```

Comment:
```suggestion
ZFS does not support standard RAID levels. It refers to virtual devices (vdevs) to describe fault tolerance within a group of devices.

In order to make the [OVHcloud API](https://api.ovh.com/) as simple as possible, customers must define a standard RAID within the API for ZFS filesystems. The standard RAID level will then be translated to an equivalent vdev definition. The following table illustrates the translation of the various RAID levels offered by the [OVHcloud API](https://api.ovh.com/) as well as a reminder of their respective characteristics.
```

And I'd throw in a link to https://openzfs.github.io/openzfs-docs/man/7/zpoolconcepts.7.html somewhere within the first paragraph.